refactor: add some notes and configurations (#1254)
GreptimeTeam/docs: docs/user-guide/deployments/deploy-on-kubernetes/getting-started.md

```
</details>

> When the `monitoring` option is enabled, the Helm chart configuration will use `mycluster` and `default` as the default cluster name and namespace.

Author: I change like this:

> When the `monitoring` option is enabled, the Helm chart configuration will use `mycluster` and `default` as the default cluster name and namespace in grafana datasources configuration.

> If you want to modify the default cluster name and namespace, you can create a `values.yaml` file like this:

Reviewer: ```suggestion
If you want to monitor clusters with a different name or a different namespace, you can create a `values.yaml` file like this:
```

```yaml
grafana:
  datasources:
    datasources.yaml:
      datasources:
        - name: greptimedb-metrics
          type: prometheus
          url: http://<cluster>-monitor-standalone.<namespace>.svc.cluster.local:4000/v1/prometheus
          access: proxy
          isDefault: true

        - name: greptimedb-logs
          type: mysql
          url: <cluster>-monitor-standalone.<namespace>.svc.cluster.local:4002
          access: proxy
          database: public
```

> 

Reviewer: please explain the code above that grafana monitoring the two clusters greptimedb-metrics and greptimedb-logs. please also explain what is the `type`.
Why the greptimedb-logs needs to specify the database public while greptimedb-metrics doesn't?

Author: I added some descriptions for the configuration. Because the Prometheus protocol doesn't have the database concept, it doesn't need to specify the database.

```
</details>

> When the `monitoring` option is enabled, we will deploy a GreptimeDB Standalone instance to store monitoring data like metrics and logs from the cluster. Additionally, we will deploy a [Vector](https://github.com/vectordotdev/vector) sidecar for each pod in the cluster to collect metrics and logs and send them to the GreptimeDB Standalone instance.

Reviewer: ```suggestion
When the `monitoring` option is enabled, a GreptimeDB Standalone instance named `<cluster-name>-monitor-standalone` will be deployed in the same namespace as the cluster to store monitoring data such as metrics and logs from the cluster. Additionally, we will deploy a [Vector](https://github.com/vectordotdev/vector) sidecar for each pod in the cluster to collect metrics and logs and send them to the GreptimeDB Standalone instance.
```

> When the `grafana` option is enabled, we will deploy a Grafana instance and configure it to use the GreptimeDB Standalone instance as a data source (using both Prometheus and MySQL protocols), allowing us to visualize the GreptimeDB cluster's monitoring data out of the box. By default, Grafana will use `mycluster` and `default` as the cluster name and namespace to create data sources. If you want to monitor clusters with different names or namespaces, you'll need to create different data source configurations based on the cluster names and namespaces. You can create a `values.yaml` file like this:

Reviewer 2: Suggestion: Add the Grafana link is [Grafana](https://grafana.com/docs/grafana/latest/).

```
</details>

> As you can see, we have created a minimal GreptimeDB cluster consisting of 1 frontend, 1 datanode, and 1 metasrv. For information about the components of a complete GreptimeDB cluster, you can refer to [architecture](/user-guide/concepts/architecture.md). Additionally, we have deployed a standalone GreptimeDB instance (`mycluster-monitor-standalone-0`) for storing monitoring data and a Grafana instance (`mycluster-grafana-675b64786-ktqps`) for visualizing the cluster's monitoring data.

Reviewer 2: we have created a minimal GreptimeDB cluster consisting of 1 frontend, 1 datanode, and 1 metasrv by default？ Added default replicas value description.
